src/features/notifications/askNotificationsModal/components/AskNotificationsModal.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FunctionComponent } from 'react'
+import styled from 'styled-components/native'
+
+import { ButtonPrimary } from 'ui/components/buttons/ButtonPrimary'
+import { AppInformationModal } from 'ui/components/modals/AppInformationModal'
+import { BicolorRingingBell } from 'ui/svg/BicolorRingingBell'
+import { Spacer, Typo } from 'ui/theme'
+import { DOUBLE_LINE_BREAK } from 'ui/theme/constants'
+
+interface Props {
+  visible: boolean
+  onHideModal: () => void
+}
+
+export const AskNotificiationsModal: FunctionComponent<Props> = ({ visible, onHideModal }) => {
+  return (
+    <AppInformationModal
+      title="Les bons plans au bon moment&nbsp;!"
+      onCloseIconPress={onHideModal}
+      visible={visible}>
+      <Spacer.Column numberOfSpaces={4} />
+      <BicolorRingingBell />
+      <Spacer.Column numberOfSpaces={4} />
+      <StyledBody>
+        {`Offres personnalisées, invitations spéciales, concours...`}
+        {DOUBLE_LINE_BREAK}
+        {`Reçois des notifications sur les bons plans du pass Culture en exclusivité\u00a0!`}
+      </StyledBody>
```

Comment:
Je pense qu'on peut aussi faire ça. A vérifier si ça s'affiche toujours bien ! (déso j'avais ps vu lors de la précédente review)

```suggestion
      <StyledBody>
        Offres personnalisées, invitations spéciales, concours...
        {DOUBLE_LINE_BREAK}
        Reçois des notifications sur les bons plans du pass Culture en exclusivité&nbsp;!
      </StyledBody>
```